src/components/DiscoveBanner/DiscoverBanner.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Button from '../Button/Button';
+import {fonts} from '../../globalStyles/fonts';
+
+const DiscoverBannerStyled = styled("div")`
+    ${fonts}
+    max-width: 1167px;
+    margin-inline: auto;
+    text-align: center;
+    background-color: rgba(255, 255, 255, 1);
+    transform: translateY(50%);
+
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+        padding-inline-start: 176px;
+        padding-inline-end: 140px;
+
+        h3 {
+            font-family: "Futura Std";
+            font-size: 50px;
+            line-heigth: 60px;
+            color: #041D42;
+            margin: 0;
+            margin-block-start: 10px;
+        }
+
+        p {
+            font-family: "Garamond";
+            font-size: 30px;
+            line-heigth: 33.75px;
+            color: #979797;
+            margin: 0;
+            margin-block-start: 63px;
+        }
+    }
+
+    @media (min-width: 900px) and (max-width: 1200px) {
+        max-width: 950px;
+        margin-inline: auto;
+    }
+
+    @media (max-width: 900px) {
+        transform: translateY(60%);
+        margin-inline-start: 11px;
+        margin-inline-end: 14px;
+        header {
+            padding-inline: 0;
+
+            h3 {
+                font-size: 30px;
+                line-height: 36px;
+                margin-block-start: 14px;
+            }
+    
+            p {
+                font-size: 20px;
+                line-height: 22.5px;
+                margin-block-start: 29px;
+            }
+        }
+    }
+`;
+
+const ButtonContainer = styled('div')`
+    padding-inline-start: 176px;
+    padding-inline-end: 140px;
+    margin-block-start: 41.24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding-block-end: 71.19px;
+
+    button {
+        font-family: "Garamond";
+        font-size: 24px;
+        line-height: 27px;
+    }
+
+    button:last-child {
+        margin-inline-start: 45px;
+    }
+
+
+    @media (max-width: 900px) {
```

Comment:
Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0